test/parallel/test-openssl-ca-options.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// This test checks the usage of --use-bundled-ca and --use-openssl-ca arguments
+// to verify that both are not used at the same time.
+const common = require('../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+const assert = require('assert');
+const os = require('os');
+const childProcess = require('child_process');
+const result = childProcess.spawnSync(process.execPath,
```

Comment:
on failure, this test will open a reply, and timeout after some time. Better  to add `-p process.version` to the spawned command line, and then fail fast if there is stdout, and succeed fast if the stderr matches expectations